ntbs-service/Models/NotificationBannerModel.cs

```diff
@@ -31,6 +31,9 @@ public class NotificationBannerModel
         public bool FullAccess;
         public string RedirectPath;
 
+        // Access level is treated as a bool for either able to edit or not. This differs from the standard PermissionLevel
+        // implemented across the codebase due to there being no visual difference between no permission level and readonly
+        // permission on notification banner models

```

Comment:
Should we change the variable name to `showPadlock` to highlight that difference and the fact that it's purely visual?